libs/home/src/lib/home.component.ts

```diff
@@ -29,8 +30,8 @@ export class HomeComponent implements OnInit {
   private readonly homeStore = inject(HomeStoreService);
   private readonly destroyRef = inject(DestroyRef);
 
-  listConfig$ = this.store.select(articleListQuery.selectListConfig);
-  tags$ = this.homeStore.tags$;
+  listConfig$: Observable<ArticleListConfig> = this.store.select(articleListQuery.selectListConfig);
```

Comment:
Interesting. Let's see why the type is not available. There should be a reason

Reply:
Actually from what I see, the type is inferred already correctly, so no need to add the type yourself

Reply:
Yes I was also surprised for this need. I'll check this once again - maybe linter error in file (separate PR) was messing things up.

I'm using Webstorm - I'll double check and revert this. Maybe Webstorm freaks out with new control flow from Angular.